src/components/PositionedOverlay/PositionedOverlay.tsx

```diff
@@ -195,12 +196,13 @@ export class PositionedOverlay extends React.PureComponent<
           onScrollOut,
           fullWidth,
           fixed,
+          preferInputActivator = true,
         } = this.props;
 
         const textFieldActivator = activator.querySelector('input');
```

Comment:
```suggestion
        const activator = preferInputActivator ? activator.querySelector('input') : activatorProp
```
Above where you deconstruct the props, I woud just rename `activator` to `activator: activatorProp`

Reply:
Wouldn't that introduce a breaking change? In it's current state `activatorRect` will always be based on an existing node. If the `input` does not exist, we use the `activator`

With you suggested change, we might end up with `null`. When `preferInputActivator` is `true`, but there's no `input`.

Since `preferInputActivator` defaults to true this would probably break most of the usages where no `TextField` is the `activator`

Reply:
yes my bad. I missed the part we do check for null today. Was trying to get away for querying the for the input if `preferInputActivator` is false.